### PR TITLE
Update SpatialSeries to have bounds field

### PR DESCRIPTION
**Pull request by mavaylon1**
## Motivation

What was the reasoning behind this change? Please explain the changes briefly.

Add test to SpatialSeries for bounds to make sure everything passes.

## How to test the behavior?
```
Show how to reproduce the new behavior (can be a bug fix or a new feature)
```

## Checklist

- [x] Did you update CHANGELOG.md with your changes?
- [x] Have you checked our [Contributing](https://github.com/NeurodataWithoutBorders/pynwb/blob/dev/docs/CONTRIBUTING.rst) document?
- [x] Have you ensured the PR clearly describes the problem and the solution?
- [x] Is your contribution compliant with our coding style? This can be checked running `flake8` from the source directory.
- [x] Have you checked to ensure that there aren't other open [Pull Requests](https://github.com/NeurodataWithoutBorders/pynwb/pulls) for the same change?
- [x] Have you included the relevant issue number using "Fix #XXX" notation where XXX is the issue number? By including "Fix #XXX" you allow GitHub to close issue #XXX when the PR is merged.

### CODE REVIEW

**Comment by mavaylon1** [comment]
Review Notes:

We talked about the idea of validating data according to the bounds (if provided); however, we decided to not validate due to performance. As result, the only test added is to ensure the bounds field is mapped correctly to the spec.

**Comment by rly** [comment]
Sorry for the late review of this @mavaylon1 . Could you add a roundtrip test with the new field?

**Comment by mavaylon1** [comment]
> Sorry for the late review of this @mavaylon1 . Could you add a roundtrip test with the new field?

Done. Sorry for the delay.

File: src/pynwb/behavior.py [hunk 1]
```diff
@@ -23,9 +23,11 @@ class SpatialSeries(TimeSeries):
     __nwbfields__ = ('reference_frame',)
 
     @docval(*get_docval(TimeSeries.__init__, 'name'),  # required
-            {'name': 'data', 'type': ('array_data', 'data', TimeSeries), 'shape': ((None, ), (None, None)),  # required
+            {'name': 'data', 'type': ('array_data', 'data', TimeSeries), 'shape': ((None, ), (None, None)), # required
              'doc': ('The data values. Can be 1D or 2D. The first dimension must be time. If 2D, there can be 1, 2, '
                      'or 3 columns, which represent x, y, and z.')},
+            {'name': 'bounds', 'type': list, 'shape': ((None, ), (None, None), (None, None, None)), 'default': None,
```
Comment (rly) [inline src/pynwb/behavior.py]:
```suggestion
            {'name': 'bounds', 'type': list, 'shape': ((1, 2), (2, 2), (3, 2)), 'default': None,
```

File: tests/integration/hdf5/test_behavior.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+from pynwb.behavior import SpatialSeries
+from pynwb.testing import AcquisitionH5IOMixin, TestCase
+
+
+class TestTimeSeriesIO(AcquisitionH5IOMixin, TestCase):
```
Comment (rly) [inline tests/integration/hdf5/test_behavior.py]:
```suggestion
class TestSpatialSeriesIO(AcquisitionH5IOMixin, TestCase):
```

File: tests/integration/hdf5/test_behavior.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    def setUpContainer(self):
+        """ Return the test TimeSeries to read/write """
+        return SpatialSeries(
+            name='test_sS',
+            data=np.ones((3, 2)),
+            reference_frame='reference_frame',
```
Comment (rly) [inline tests/integration/hdf5/test_behavior.py:15]:
```suggestion
            bounds=[(-1,1),(-1,1),(-1,1)],
            reference_frame='reference_frame',
```

**Review by rly — APPROVED** [review]

**Pull request merged**